src/plans/details/tabs/Mappings/components/PlanNetworkMapEdit/components/TargetNetworkField.tsx

```diff
@@ -0,0 +1,58 @@
+import type { FC } from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+import type { NetworkMappingId } from 'src/networkMaps/utils/getNetworkMapFieldId';
+import type { MappingValue } from 'src/plans/create/types';
+import { IgnoreNetwork } from 'src/plans/details/tabs/Mappings/utils/constants';
+
+import Select from '@components/common/Select';
+import { Divider, SelectList, SelectOption } from '@patternfly/react-core';
+import { useForkliftTranslation } from '@utils/i18n';
+
+import type { PlanNetworkEditFormValues } from '../utils/types';
+
+type TargetNetworkFieldProps = {
+  fieldId: NetworkMappingId;
+  targetNetworks: Record<string, MappingValue>;
+};
+
+const TargetNetworkField: FC<TargetNetworkFieldProps> = ({ fieldId, targetNetworks }) => {
```

Comment:
There are 3 components that  at least 2 of them look identical, one for create and one for edit plans and the 3d for network maps editing:

-` networkMaps/create/fields/TargetNetworkField`
-` plans/create/steps/network-map/TargetNetworkField`
-`plans/details/tabs/Mappings/components/PlanNetworkMapEdit/components TargetNetworkField`

Can we combine them and use one component for all?